packages/compiler/src/__tests__/__snapshots__/typescript.test.ts.snap

```diff
@@ -1067,25 +868,18 @@ Object {
       "isStatic": false,
     },
     "implements": undefined,
-    "indexSignature": Object {
-      "documentation": Object {
-        "contents": Array [
-          "<p>Index signature for the masses.</p>
-",
-        ],
-        "contentsRaw": "Index signature for the masses.",
-        "metadata": Object {},
-      },
+    "indexSignature": {
+      "documentation": undefined,
```

Comment:
this index signature jsdoc got dropped - not sure why 🤔

Reply:
I believe this is a bug in JSDoc, filed this issue: https://github.com/TypeStrong/typedoc/issues/2414

But I don't think we need to block on this; this is an uncommonly used feature.